packages/go_router/pubspec.yaml

```diff
@@ -1,13 +1,12 @@
 name: go_router
 description: A declarative router for Flutter based on Navigation 2 supporting
   deep linking, data-driven routes and more
-version: 3.1.1
+version: 4.0.0
 repository: https://github.com/flutter/packages/tree/main/packages/go_router
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+go_router%22
 
 environment:
   sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=2.0.0"
```

Comment:
shouldn't the package depend on flutter 3.0.0 ? i am getting `WidgetsBinding.instance` warnings
because of `static WidgetsBinding get _binding => WidgetsBinding.instance!;`